jwst/guider_cds/guider_cds.py

```diff
@@ -98,6 +129,72 @@ def guider_cds(model):
     return new_model
 
 
+def get_ref_arr(model, imshape, gain_model=None, readnoise_model=None):
+    """
+    Short Summary
+    -------------
+    Create gain and readnoise files from their reference files
```

Comment:
This function does not create reference files. It extracts data arrays in appropriate shape for the sci data. The one sentence short summary description needs to be corrected.